src/eradius_proxy.erl

```diff
@@ -111,8 +136,15 @@ decode_request(Result, ReqID, Secret, Auth) ->
     end.
 
 % @private
--spec validate_route({Host :: list() | binary() | inet:ip_address(), Port :: eradius_server:port_number(), Secret :: eradius_lib:secret()}) ->
-    boolean().
+-spec validate_route({Host :: list() | binary() | inet:ip_address(),
```

Comment:
Maybe have -type() for these two arguments?

Reply:
Yes, but it looks like it could be just replaced with `route` type here

Reply:
this is fixed now